src/coreclr/System.Private.CoreLib/src/System/Threading/Thread.CoreCLR.cs

```diff
@@ -8,6 +8,12 @@
 using System.Runtime.Serialization;
 using System.Runtime.Versioning;
 
+namespace System.Runtime
+{
+    internal sealed class BypassReadyToRunAttribute : Attribute
+    {
+    }
+}
```

Comment:
I would create a new file for this. It is oddly placed and doesn't follow the guidelines since there are now two non-nested namespaces in this file.